README.md

```diff
@@ -4,39 +4,39 @@
 Postfacto helps teams run great [retrospecives](https://content.pivotal.io/blog/how-to-run-a-really-good-retrospective) remotely.
 
 <p align="center">
-  <img width="700px" src="https://github.com/crswty/postfacto/blob/feat-new-readme/media/sample-retro.gif?raw=true" />
+  <img width="700px" src="https://github.com/pivotal/postfacto/blob/master/media/sample-retro.gif?raw=true" />
 </p>
 
 
 ## Features
 
 ### Run Retros Remotely
-Postfacto lets you run an agile retrospective even when your team is distributed across the world. Retros live update across all your devices
-so each participant can follow along on their device.
+Postfacto lets you run an agile retrospective even when your team is distributed across the world. Your retros will live update across all your devices so each participant can follow along on their device.
 
-### Record Action Items
-Retros are designed to help the team improve and that's hard to do without keeping track of the actions the team needs to carry 
-out and the also the one's you've achieved. Postfacto tracks the items you enter week to week to help you keep on top of them.
+### Easy Onboarding
+Easily [set up](https://github.com/pivotal/postfacto/tree/master/deployment#allowing-users-to-create-retros) Postfacto to work with Google OAuth so users can sign up with their Google accounts. Alternatively, you can control access to your instance with the admin dashboard.
+
+### Define Retro Access
+You can create private retro boards for your team that are password protected or choose to leave them public so that anyone you give the link to can access them without friction.
```

Comment:
```suggestion
You can create private retro boards for your team that are password protected or choose to leave them public so that anyone you give the link to can access them.
```